Sources/ApolloCodegenLib/Templates/SelectionSetTemplate.swift

```diff
@@ -5,12 +5,20 @@ struct SelectionSetTemplate {
 
   let schema: IR.Schema
   let isMutable: Bool
+  let config: ReferenceWrapped<ApolloCodegenConfiguration>
```

Comment:
I know we spoke about getting rid of `ReferenceWrapped`, and it is not used in some parts of this PR, but I think that should go into a separate PR and tackle it throughout the codegen codebase.